src/app/results/results.component.spec.ts

```diff
@@ -123,6 +127,12 @@ describe('ResultsComponent', () => {
     expect(compiled.querySelector('app-theme')).toBeTruthy();
   });
 
+  it('should have an app-infobox element', () => {
+        let compiled = fixture.debugElement.nativeElement;
```

Comment:
Always use ```const``` instead of ```let```

Reply:
Currently `let` is used in all `spec` files in project and at some places it throws an error while using `const` . For `spec` files I will go through it in detail and wherever possible I will replace `let` with `const` in a separate PR :smile:

Reply:
The error is in deploying? or it is just shown on vscode? Is there any typescript error? And also add linting tests similar to loklak which tests all these cases.

Reply:
Make changes related to ```let``` in all places in another patch since this patch correspondingly solves another issue.